# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1472244997 14400
#      Fri Aug 26 16:56:37 2016 -0400
# Node ID c5b9c202e637d7e0cf071d5d5b5eaf303ad2ccb6
# Parent  5e9e4d4a0d3331547f9007a2dd40e4f21454e875
* cmd-hist.cc (command_history::do_process_histcontrol):
Don't warn if history control is not available.

diff --git a/liboctave/util/cmd-hist.cc b/liboctave/util/cmd-hist.cc
--- a/liboctave/util/cmd-hist.cc
+++ b/liboctave/util/cmd-hist.cc
@@ -768,21 +768,17 @@ namespace octave
   command_history::clean_up_and_save (const std::string& f, int n)
   {
     if (instance_ok ())
       instance->do_clean_up_and_save (f, n);
   }
 
   void
   command_history::do_process_histcontrol (const std::string&)
-  {
-    (*current_liboctave_warning_with_id_handler)
-      ("Octave:history-control",
-       "readline is not linked, so history control is not available");
-  }
+  { }
 
   void
   command_history::do_initialize (bool read_history_file,
                                   const std::string& f_arg, int sz,
                                   const std::string & control_arg)
   {
     command_history::set_file (f_arg);
     command_history::set_size (sz);
